<ui version="4.0" >
 <class>SceneEditor</class>
 <widget class="QDialog" name="SceneEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Scene editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="1" column="1" >
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="m_updateValuesCheck" >
     <property name="toolTip" >
      <string>As sliders are dragged, send their data to the actual fixtures to see what really happens</string>
     </property>
     <property name="text" >
      <string>Send slider data to fixtures</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QTabWidget" name="m_tab" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="General" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>532</width>
        <height>249</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <widget class="QLabel" name="m_nameLabel" >
           <property name="text" >
            <string>Scene name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_nameEdit" >
           <property name="toolTip" >
            <string>Name of this scene</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <widget class="QTreeWidget" name="m_tree" >
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <property name="itemsExpandable" >
            <bool>false</bool>
           </property>
           <property name="sortingEnabled" >
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus" >
            <bool>true</bool>
           </property>
           <property name="expandsOnDoubleClick" >
            <bool>false</bool>
           </property>
           <column>
            <property name="text" >
             <string>Fixtures used in this scene</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QToolButton" name="m_addFixtureButton" >
             <property name="toolTip" >
              <string>Add a new fixture to this scene</string>
             </property>
             <property name="text" >
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_removeFixtureButton" >
             <property name="toolTip" >
              <string>Remove the selected fixture(s) from this scene</string>
             </property>
             <property name="text" >
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="All" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>532</width>
        <height>249</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>All</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SceneEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>158</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SceneEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>361</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel" >
     <x>308</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
